### PR TITLE
[Gogoanime presence] fixed subdomain issue

### PR DESCRIPTION
**Pull request by PsychoPast**

### CODE REVIEW

File: websites/G/Gogoanime/presence.ts [hunk 1]
```diff
@@ -92,19 +97,30 @@ async function sendRequestToDomainAPI(): Promise<Response> {
...
 
-async function checkDomain(): Promise<boolean> {
-  const cookies = parseCookieString(document.cookie),
-    currentDomain = document.location.hostname,
-    cookieName = "PMD_GOGOANIME_VALID_DOMAINS";
+async function checkDomain(): Promise<DomainCheckState> {
+  const result: DomainCheckState = {invalid: true, validDomains: null},
+       cookies = parseCookieString(document.cookie),
+    cookieName = "PMD_GOGOANIME_VALID_DOMAINS",
+    domainParts = document.location.hostname.split('.');
```
Comment (EncryptedDev) [inline websites/G/Gogoanime/presence.ts:104]: prettier?

Comment (Bas950) [reply websites/G/Gogoanime/presence.ts:104]: SeMiD will do it

**Review by Bas950 — APPROVED** [review]
LGTM

**Comment by ririxidev** [comment]
Screenshots pls

**Comment by PsychoPast** [comment]
> Screenshots pls

sure sorry I forgot
image
image
image

File: websites/G/Gogoanime/dist/metadata.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "gogoanime.news"
   ],
   "regExp": "[0-9w]{0,6}.*gogoanime[s]*.[a-z]+",
-  "version": "2.0.3",
+  "version": "2.0.5",
```
Comment (Bas950) [inline websites/G/Gogoanime/dist/metadata.json:30]: 1 version bump is enough.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Gogoanime/dist/metadata.json (hunk 2 above)
Comment (PsychoPast) [reply websites/G/Gogoanime/dist/metadata.json]: done

**Review by ririxidev — APPROVED** [review]

**Comment by ririxidev** [comment]
oops something broke with your fork, can you refork it with your changes and create new pull request ty

**Pull request closed without merging**